src/3d/qgsgltf3dutils.cpp

```diff
@@ -100,9 +100,18 @@ static Qt3DRender::QTextureWrapMode::WrapMode parseTextureWrapMode( int wrapMode
 {
   switch ( wrapMode )
   {
-    case TINYGLTF_TEXTURE_WRAP_REPEAT: return Qt3DRender::QTextureWrapMode::Repeat;
-    case TINYGLTF_TEXTURE_WRAP_CLAMP_TO_EDGE: return Qt3DRender::QTextureWrapMode::ClampToEdge;
-    case TINYGLTF_TEXTURE_WRAP_MIRRORED_REPEAT: return Qt3DRender::QTextureWrapMode::MirroredRepeat;
+    case TINYGLTF_TEXTURE_WRAP_REPEAT:
+      return Qt3DRender::QTextureWrapMode::Repeat;
+    case TINYGLTF_TEXTURE_WRAP_CLAMP_TO_EDGE:
+      return Qt3DRender::QTextureWrapMode::ClampToEdge;
+    case TINYGLTF_TEXTURE_WRAP_MIRRORED_REPEAT:
+      return Qt3DRender::QTextureWrapMode::MirroredRepeat;
+
+    // non standard cases:
+    case 10242:
+      // seen in https://qld.digitaltwin.terria.io/api/v0/data/b73ccb60-66ef-4470-8c3c-44af36c4d69b/CBD/tileset.json
+      return Qt3DRender::QTextureWrapMode::Repeat;
```

Comment:
I would suggest not to have a special case for this, and just remove the Q_ASSERT below, so it will get the Repeat wrap mode as if the wrap mode was not specified in the file (which is the default value given by gltf spec).

I have had a look and this is the offending bit in the GLTF:
```json
  "samplers": [
    {
      "magFilter": 9729,
      "minFilter": 9729,
      "wrapS": 10242,
      "wrapT": 10243
    }
  ],
```
Neither "wrapS" nor "wrapT" have valid values - it looks like a badly implemented GLTF writer as those values coincide with these OpenGL values:
```cpp
#define GL_TEXTURE_WRAP_S 0x2802
#define GL_TEXTURE_WRAP_T 0x2803
```
I am actually surprised this fixed it for you, given that the value 10243 should trigger the assert as well...

It is also confirmed by gltf validator (and there are more issues with these GLTF files):
![image](https://github.com/qgis/QGIS/assets/193367/ee10a2d1-df67-4507-a32e-771e352eaedd)

(by the way - this is a really handy little app which includes the validator: https://gltf-viewer.donmccurdy.com/)

Reply:
Ok, done